scene/gui/line_edit.cpp

```diff
@@ -539,7 +539,7 @@ void LineEdit::_gui_input(Ref<InputEvent> p_event) {
 
 			if (handled) {
 				accept_event();
-			} else if (!k->get_alt() && !k->get_command()) {
+			} else if (!k->get_command()) {
```

Comment:
It would be good to check why `!k->get_alt()` was used previously and if this change would introduce issues on non-macOS. From a quick look I see above that in some cases `handled` is set to `false` for `k->get_alt()` on non-Apple, so this would now fall into this branch.

Reply:
This check was added in #5271, not sure what was it supposed to fix.

On Windows all `Alt+?` seems to have `unicode` set to `0` and `AltGr` combinations with set `unicode` have `set_alt` flag removed. I have not found any difference in input behaviour.

On Linux (Debian, Xfce) `Alt+?` have `unicode` values set, and act as text input (without this PR Alt+? input was not accepted by LineEdit, but was by TextEdit, behaviour of other apps seems to be inconsistent). `AltGr` combinations doesn't have `set_alt` flag set.

Just in case I'm missing something I have added extra check for the cases setting `handled` to `false`.

Reply:
I trust your research and tests, I don't think we should add extra checks which are not particularly obvious to the reader. Either we know that they are really needed and we should put a comment explaining why, or it's better to leave them out and we can reconsider if anyone reports a regression when using Alt + one of those keys on a given platform.